config/production.yml.erb

```diff
@@ -11,7 +11,6 @@ redshift_host: !Secret
 redshift_password: !Secret
 redshift_username: !Secret
 sinatra_session_secret: !Secret
-slog_token: !Secret
```

Comment:
I'm guessing there is cleanup to do in our secrets manager, but it shouldn't be breaking to remove this here.